src/qt/locale/bitcoin_id_ID.ts

```diff
@@ -1,1871 +1,5008 @@
-<TS language="id_ID" version="2.1">
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Need to fix this, remove xml tags should remain as
`<TS language="id_ID" version="2.1">`